packages/angular_devkit/build_angular/src/utils/process-bundle.ts

```diff
@@ -149,13 +156,21 @@ async function processWorker(options: ProcessBundleOptions): Promise<void> {
       code += `\n//# sourceMappingURL=${path.basename(newFilePath)}.map`;
     }
 
+    if (options.cacheKey && options.cachePath) {
+      const key = options.cacheKey + '-dl-map' + (options.vendorSourceMaps ? '-v' : '');
+      await cacache.put(options.cachePath, key, map);
+    }
+
     fs.writeFileSync(newFilePath + '.map', map);
   }
 
+  if (options.cacheKey && options.cachePath) {
+    await cacache.put(options.cachePath, options.cacheKey + '-dl', code);
```

Comment:
Why not provide the full `cacheKey` when invoking? Rather than just the hash to avoid building the cache key twice?

IE:

consumer
```js
const cacheKey = `${packageVersion}-${action.code.length}-${codeHash}-dl`;
processActions.push({
	...action,
	cacheKey,
	...
});
```
here:
```
await cacache.put(options.cachePath, options.cacheKey, code);
```